static/src/stylesheets/content.scss

```diff
@@ -118,6 +118,7 @@
    ========================================================================== */
 
 @import 'module/experiments/_affix';
+@import 'module/experiments/_membership';
```

Comment:
I know that I am a little bit picky right now, but I would rename the file to eg `_membership-test` os sth like that. The `membership` is too general.